### PR TITLE
Config flow opt

### PR DESCRIPTION
**Pull request by AaronDavidSchneider**
optimizing config flow
Already fixed: 
- [x] Multistep configuration

### CODE REVIEW

**Comment by mammuth** [comment]
Wow, cool! I will check it out in detail!

File: custom_components/fritzbox_tools/.translations/de.json [hunk 1]
```diff
@@ -1,27 +1,33 @@
 {
     "config": {
-        "title": "FRITZ!Box Tools",
+        "title": "Fritz!Box Tools",
```
Comment (mammuth) [inline custom_components/fritzbox_tools/.translations/de.json]: Were these naming changes intentional or by (accident)? Do you think we should call it `Fritz!Box Tools`?

**Comment by mammuth** [comment]
It seems to work fine for me. Really like this. It makes it much easier to use.

File: custom_components/fritzbox_tools/.translations/de.json (hunk 1 above)
Comment (AaronDavidSchneider) [reply custom_components/fritzbox_tools/.translations/de.json]: Oh I think this wasn’t intentional. Sorry...

Comment (mammuth) [reply custom_components/fritzbox_tools/.translations/de.json]: Never mind, just wanted to be sure about it! I'll rename it separately.

**Pull request merged**